src/plugins/rename/__init__.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
+
+import nonebot
+from nonebot import require, logger, get_bot
+
+from src.plugins.repeater.model import Chat
+
+change_name_sched = require('nonebot_plugin_apscheduler').scheduler
+
+
+@change_name_sched.scheduled_job('cron', hour='*/1')
+async def change_name():
+    ret = Chat.get_change_name_id()
+    if not ret:
+        return
+    group_target_list, bot_id = ret
+    for group_id, target_id in group_target_list:
+        logger.info(
+                'rename | bot [{}] ready to change name by using [{}] in group [{}]'.format(
+                    bot_id, target_id, group_id))
+        # 获取群友昵称
+        info = await get_bot(str(bot_id)).call_api('get_group_member_info', **{
+                'group_id': group_id,
+                'user_id': target_id,
+                'no_cache': True
+            })
+        # 有时会获取不了群友的群昵称，不知道为啥
+        await asyncio.sleep(5)
+        card = info['card']

```

Comment:
```suggestion
        card = info['card']
        card = info['card'] if info['card'] else info['nickname']
```

获取不了应该是没设置群昵称，这时候可以用 QQ 昵称（或者直接跳过这个人？）

上面一行的 sleep 是为了这个吗，如果是可以删掉看看